### PR TITLE
Bump Traefik to v2.0.2

### PR DESCRIPTION
**Pull request by ldez**
Hello,

This PR updates Traefik to [v2.0.2](https://github.com/containous/traefik/releases/tag/v2.0.2).

/cc @mmatur @emilevauge @dtomcej

Cheers

### CODE REVIEW

File: library/traefik [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-GitCommit: e158b458340bad5e6c039ba84a4768e0d7e2195f
+GitCommit: a232b9ebed96febf723daa43c6899ec86ba554eb
 Directory: windows/1809
 Constraints: windowsservercore-1809
 
-Tags: v2.0.1, 2.0.1, v2.0, 2.0, montdor, latest
+Tags: v2.0.2, 2.0.2, v2.0, 2.0, montdor, latest
 Architectures: amd64, arm32v6, arm64v8
-GitCommit: e158b458340bad5e6c039ba84a4768e0d7e2195f
+GitCommit: a232b9ebed96febf723daa43c6899ec86ba554eb
```
Comment (tianon) [inline library/traefik]: These should both be `3d06252965f0028af9de0787b8caa4a1623a9e96`, right? (`a232b9ebed96febf723daa43c6899ec86ba554eb` doesn't seem to exist on any branches in https://github.com/containous/traefik-library-image)

Comment (ldez) [reply library/traefik]: we had a little problem with the publication of the release binaries, I had to make a push force and redo the release of the version.

Comment (ldez) [reply library/traefik]: The problem is fixed now.

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index ea1270a..4159a04 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,8 +6,8 @@ traefik:1.7.18-alpine
 traefik:1.7.18-windowsservercore-1809
 traefik:2.0
 traefik:2.0-windowsservercore-1809
-traefik:2.0.1
-traefik:2.0.1-windowsservercore-1809
+traefik:2.0.2
+traefik:2.0.2-windowsservercore-1809
 traefik:latest
 traefik:maroilles
 traefik:maroilles-alpine
@@ -22,6 +22,6 @@ traefik:v1.7.18-alpine
 traefik:v1.7.18-windowsservercore-1809
 traefik:v2.0
 traefik:v2.0-windowsservercore-1809
-traefik:v2.0.1
-traefik:v2.0.1-windowsservercore-1809
+traefik:v2.0.2
+traefik:v2.0.2-windowsservercore-1809
 traefik:windowsservercore-1809
diff --git a/traefik_latest/Dockerfile b/traefik_latest/Dockerfile
index 3360bba..a13255a 100644
--- a/traefik_latest/Dockerfile
+++ b/traefik_latest/Dockerfile
@@ -8,7 +8,7 @@ RUN set -ex; \
 		x86_64) arch='amd64' ;; \
 		*) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
 	esac; \
-	wget --quiet -O /tmp/traefik.tar.gz "https://github.com/containous/traefik/releases/download/v2.0.1/traefik_v2.0.1_linux_$arch.tar.gz"; \
+	wget --quiet -O /tmp/traefik.tar.gz "https://github.com/containous/traefik/releases/download/v2.0.2/traefik_v2.0.2_linux_$arch.tar.gz"; \
 	tar xzvf /tmp/traefik.tar.gz -C /usr/local/bin traefik; \
 	rm -f /tmp/traefik.tar.gz; \
 	chmod +x /usr/local/bin/traefik
@@ -22,5 +22,5 @@ LABEL org.opencontainers.image.vendor="Containous" \
 	org.opencontainers.image.url="https://traefik.io" \
 	org.opencontainers.image.title="Traefik" \
 	org.opencontainers.image.description="A modern reverse-proxy" \
-	org.opencontainers.image.version="v2.0.1" \
+	org.opencontainers.image.version="v2.0.2" \
 	org.opencontainers.image.documentation="https://docs.traefik.io"
diff --git a/traefik_windowsservercore-1809/Dockerfile b/traefik_windowsservercore-1809/Dockerfile
index 75b2464..e3277f9 100644
--- a/traefik_windowsservercore-1809/Dockerfile
+++ b/traefik_windowsservercore-1809/Dockerfile
@@ -3,7 +3,7 @@ FROM mcr.microsoft.com/windows/servercore:1809
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 RUN Invoke-WebRequest \
-        -Uri "https://github.com/containous/traefik/releases/download/v2.0.1/traefik_v2.0.1_windows_amd64.zip" \
+        -Uri "https://github.com/containous/traefik/releases/download/v2.0.2/traefik_v2.0.2_windows_amd64.zip" \
         -OutFile "/traefik.zip"; \
     Expand-Archive -Path "/traefik.zip" -DestinationPath "/" -Force; \
     Remove-Item "/traefik.zip" -Force
@@ -16,5 +16,5 @@ LABEL org.opencontainers.image.vendor="Containous" \
     org.opencontainers.image.url="https://traefik.io" \
     org.opencontainers.image.title="Traefik" \
     org.opencontainers.image.description="A modern reverse-proxy" \
-    org.opencontainers.image.version="v2.0.1" \
+    org.opencontainers.image.version="v2.0.2" \
     org.opencontainers.image.documentation="https://docs.traefik.io"
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #6752; fcdfce05; `amd64` (`traefik`):

```console

$ bashbrew build traefik:v2.0.2-windowsservercore-1809
skipping "traefik:v2.0.2-windowsservercore-1809" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh traefik:v2.0.2-windowsservercore-1809
testing traefik:v2.0.2-windowsservercore-1809
	image has no tests...skipping


$ bashbrew build traefik:v2.0.2
Building bashbrew/cache:35c1bb53e287114e1352bccee3d313311278e50a22013bd661499e19636814a3 (traefik:v2.0.2)
Tagging traefik:v2.0.2
Tagging traefik:2.0.2
Tagging traefik:v2.0
Tagging traefik:2.0
Tagging traefik:montdor
Tagging traefik:latest

$ test/run.sh traefik:v2.0.2
testing traefik:v2.0.2
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...= 'echo' is not a Traefik command: assuming shell execution.
passed


$ bashbrew build traefik:v1.7.18-windowsservercore-1809
skipping "traefik:v1.7.18-windowsservercore-1809" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh traefik:v1.7.18-windowsservercore-1809
testing traefik:v1.7.18-windowsservercore-1809
	image has no tests...skipping


$ bashbrew build traefik:v1.7.18-alpine
Using bashbrew/cache:4367f29441f3d0ef63a537ddc79e3ba4e5f858f60d9106436b17e1c0a3aa21ab (traefik:v1.7.18-alpine)
Tagging traefik:v1.7.18-alpine
Tagging traefik:1.7.18-alpine
Tagging traefik:v1.7-alpine
Tagging traefik:1.7-alpine
Tagging traefik:maroilles-alpine

$ test/run.sh traefik:v1.7.18-alpine
testing traefik:v1.7.18-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...= 'echo' is not a Traefik command: assuming shell execution.
passed


$ bashbrew build traefik:v1.7.18
Using bashbrew/cache:65bd2245b32035f0b636c25d38489b8a33a5bc3ffd366163266baed615f5dd86 (traefik:v1.7.18)
Tagging traefik:v1.7.18
Tagging traefik:1.7.18
Tagging traefik:v1.7
Tagging traefik:1.7
Tagging traefik:maroilles

$ test/run.sh traefik:v1.7.18
testing traefik:v1.7.18
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed

```

**Pull request merged**